### PR TITLE
Update style of the documentation front page

### PR DESCRIPTION
**Pull request by st-pasha**
The new design is more energetic and fun; and it also better resonates with the style of our logo.

## New design
Screen Shot 2020-09-22 at 7 21 22 PM

## Old design
Screen Shot 2020-09-22 at 7 22 33 PM

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -1,50 +1,155 @@
...
-          <div><a class="reference internal" href="develop/index-develop.html">
-           &#x25AA; Development
-          </a></div>
-          <div><a class="reference internal" href="genindex.html">
-           &#x25AA; Index
-          </a></div>
-        </div>
+    <div id="dtcontainer">
+      <div id="description"><p>
+        <b>Datatable</b> is a python library for manipulating tabular data.
```
Comment (oleksiyskononenko) [inline docs/index.rst:114]: I understand that this is a first word in the sentence, but still, shouldn't it be `datatable` instead of `Datatable`?

**Comment by oleksiyskononenko** [comment]
Looks nice, have you had time to check that it renders ok in all the major browsers?

**Review by oleksiyskononenko — APPROVED** [review]

File: docs/index.rst (hunk 1 above)
Comment (st-pasha) [reply docs/index.rst:114]: Yeah, I tried that at first, but it just doesn't look as good

Comment (oleksiyskononenko) [reply docs/index.rst:114]: May be we unify how `datatable` word is spelled and looks everywhere on the docs?

**Comment by st-pasha** [comment]
Works in Chrome, Safari and Firefox. I don't have access to IE/Edge.

**Comment by oleksiyskononenko** [comment]
Just checked it in Edge and it looks good.

**Pull request merged**